packages/asset-service/src/generateAssetData/baseAssets.ts

```diff
@@ -148,3 +148,16 @@ export const bnbsmartchain: Asset = {
   explorerAddressLink: 'https://bscscan.com/address/',
   explorerTxLink: 'https://bscscan.com/tx/',
 }
+
+export const polygon: Asset = {
+  assetId: caip.polygonAssetId,
+  chainId: caip.polygonChainId,
+  name: 'Polygon',
+  symbol: 'MATIC',
+  precision: 18,
+  color: '#8f00ff',
+  icon: 'https://assets.coingecko.com/coins/images/4713/large/matic-token-icon.png?1624446912',
```

Comment:
Confirmed that's better, weird PNG with a white bg but that's upstream so we can't control it